packages/code/src/renderingPlugin.tsx

```diff
@@ -61,22 +82,20 @@ const cellStyle = css`
   border-spacing: 0;
   padding: 0;
   vertical-align: top;
-
-  &:first-of-type {
-    padding-left: 12px;
-  }
-
-  &:last-of-type {
-    padding-right: 12px;
-  }
+  padding-right: ${spacing[3]}px;
+  padding-left: ${spacing[3]}px;
```

Comment:
we can collapse some of these
`padding: 0 ${spacing[3]}px;`

Reply:
Oh you're right, I have `padding: 0` above. Nice catch.